driver-core/src/main/java/com/datastax/driver/core/GettableByIndexData.java

```diff
@@ -90,6 +90,30 @@
     public Date getDate(int i);
 
     /**
+     * Returns the {@code i}th value as a date as an int in days since epoch.
+     *
+     * @param i the index ({@code 0 <= i < size()}) to retrieve.
+     * @return the value of the {@code i}th element as a data. If the
+     * value is NULL, {@code 0} is returned.
+     *
+     * @throws IndexOutOfBoundsException if {@code i} is not a valid index for this object.
+     * @throws InvalidTypeException if value {@code i} is not of type DATE.
+     */
+    public int getSimpleDate(int i);
```

Comment:
I think there is an agreement to break compatibility and call this getDate() while the former getDate() becomes getTimestamp()?